hybridq/architecture/ibm/eagle.py

```diff
@@ -0,0 +1,83 @@
+"""
```

Comment:
Just a comment- this file will be tough to maintain. Is the information present in ibm's software at all?

Reply:
One could in principle use `pyquil.api.get_qc`, but not all the architectures are present (at the moment, `pyquil.api.list_quantum_computers()` provides only `Aspen-11` for me) and it's hard to get the nice 2D layout from the APIs provided by the different quantum companies (for instance, `pyquil` only provides a `networkx` of the layout).

On the other hand, once a name is given to a QPU, in this case `Eagle`, it's very unlikely that the manufacturer will change the layout. We can plan to have a better way to retrieve QPU layouts in the future.

Reply:
Opened a new issue: #94